src/shared/download-file.ts

```diff
@@ -0,0 +1,49 @@
+function downloadFile(
+  content: string,
+  type: string,
+  filename: string,
+  document: Document,
+): void {
+  const blob = new Blob([content], { type });
+  const url = URL.createObjectURL(blob);
+
+  const link = document.createElement('a');
+  link.setAttribute('href', url);
+  link.setAttribute('download', filename);
+  link.style.visibility = 'hidden';
+
+  document.body.appendChild(link);
+  link.click();
+  document.body.removeChild(link);
+}
```

Comment:
This is was previously part of `downloadJSONFile`, but it is logic we can reuse to download any file.